cargo-verify/src/seahorn.rs

```diff
@@ -77,40 +77,21 @@ fn importance(line: &str, expect: &Option<&str>, name: &str) -> i8 {
 
 /// Run Seahorn and analyse its output.
 fn run(opt: &Opt, name: &str, entry: &str, bcfile: &Path, out_dir: &Path) -> CVResult<Status> {
+    let verify_common_dir = match &opt.verify_common_dir {
+        Some(verify_common_dir) => verify_common_dir,
+        None => Err("The '--verify_common_dir' option is missing")?,
+    };
+
     let mut cmd = Command::new("sea");
-    cmd.args(&["bpf",
```

Comment:
Ah, this looks like what I was asking for in the other, separated commit!
Moving this content into an external file.